docs/examples/scheduled-capacity.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: autoscaling.karpenter.sh/v1alpha1
+kind: MetricsProducer
+metadata:
+  name: scheduling
+spec:
+  scheduleSpec:
+    timezone: America/Los_Angeles
+    defaultReplicas: 1
+    behaviors:
+    # leaving hours/minutes empty == specifying 0
+    # leaving every other field empty == *
+    # single value (non-replicas) int fields need quotations
```

Comment:
When YAML sees an integer by itself, it assumes that the field is an int, while it should be a string. Any non-singular value will have a comma, so it will be recognized as a string otherwise.

Reply:
ok, I see we have them as string. 
This comment is confusing, since we are not parsing them as ints, we shouldn't mention it here. Just having quotes means we expect input as a string, it shouldn't matter if it's single or multiple digits.